lib/elixir/test/elixir/typespec_test.exs

```diff
@@ -907,6 +907,26 @@ defmodule TypespecTest do
       end)
     end
 
+    test "fails with a helpful error when non-variables are given as arguments" do
```

Comment:
Maybe `test "non-variables are given as arguments" do` would read equally good. Saying "helpful" is quite subjective.